sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -201,6 +201,19 @@ object SQLConf {
     .stringConf
     .createOptional
 
+  val PLAN_HINT_IGNORE_LOG_LEVEL =
```

Comment:
Does this really worth a config?

Reply:
+1. I don't think we need to make it configurable. We can just log warning.